src/_gettsim/transfers/rente.py

```diff
@@ -968,3 +1109,28 @@ def ges_rente_anrechnungszeit_45(
     out = m_arbeitsunfähig + m_alg1_übergang + m_geringf_beschäft
 
     return out
+
+
+def birthdate_decimal(
```

Comment:
This function should live in `demographic_vars.py`

Reply:
moved